bluesky/tool.gpt

```diff
@@ -0,0 +1,43 @@
+---
+Name: Bluesky
+Metadata: bundle: true
+Description: Tools for interacting with Bluesky (bsky.app)
+Share Tools: Search Posts, Search Users
+
+---
+Name: Search Posts
+Description: Search for posts on Bluesky
+JSON Response: true
+Share Context: ../time
+Tools: service
+Param: query: A Lucene query to search for posts with.
+Param: since: (optional) ISO 8601 timestamp to search for posts since. Defaults to 7 days ago.
+Param: until: (optional) ISO 8601 timestamp to search for posts until. Defaults to now.
+Param: limit: (optional) The maximum number of posts to return. Must be an integer >=1 and <=100. Defaults to 25.
+
+#!http://service.daemon.gptscript.local/tools/posts/search
+
+---
+Name: Search Users
+Description: Search for users on Bluesky
+JSON Response: true
+Share Context: ../time
+Tools: service
+Param: query: A Lucene query to search for users with.
+Param: limit: (optional) The maximum number of users to return. Must be an integer >=1 and <=100. Defaults to 25.
+
+#!http://service.daemon.gptscript.local/tools/users/search
+
+---
+Name: service
+Metadata: index: false
+
+#!sys.daemon /usr/bin/env npm --prefix ${GPTSCRIPT_TOOL_DIR} run server
```

Comment:
Interesting that you chose to do this as a daemon. I don't think this works when we add authentication.

Reply:
Oh, I had assumed the creds would get passed with the env header. I'll switch to a non-daemon

Reply:
OK -- updated this to be a "command driven" tool